frontend/src/app/components/nav-bar/nav-bar.component.html

```diff
@@ -6,7 +6,7 @@
         </ng-container>
         <ng-template #app_brand >
             <img class="app-navbar-brand__logo" src="assets/icon/medtagger_logo.svg" alt="MedTagger logo" />
-            <span class="app-navbar-brand__text">MedTagger</span>
+            <span class="app-navbar-brand__text">{{ 'APP_NAME' | translate }}</span>
```

Comment:
I don't think that app name would ever be translated...

Reply:
Just a placeholder to bind all occurrences of 'MedTagger' phrase.